Gemfile.lock

```diff
@@ -342,9 +349,9 @@ GEM
       net-ssh (>= 2.6.5)
     net-ssh (3.0.2)
     netrc (0.10.3)
-    nio4r (1.2.1)
-    nokogiri (1.7.2)
+    nio4r (2.0.0)
       mini_portile2 (~> 2.1.0)
+    nokogiri (1.7.2)
```

Comment:
You are right, `mini_portile2` is a nokogiri dependency 😄 I am surprised bundler did not complain about it or fix the Gemfile.lock